suite/tests/CMakeLists.txt

```diff
@@ -2646,6 +2646,10 @@ if (NOT ANDROID) # TODO i#38: Port test to Android.
     torunonly_ci(client.attach_blocking linux.infloop client.attach_test.dll
       client-interface/attach_blocking.runall "" "" "")
   endif ()
+  if (LINUX AND X64)
```

Comment:
If this API works on Windows can we enable the test there too?  (If it's bit-rotted and fails, add a TODO here about enabling on Windows in the future...maybe file a new issue on it too.)

Reply:
The same test runs fine on vs2019-64 but crashes on vs2019-32. I've added to test to vs2019-64, and filed a feature request #7071 for v2019-32.